src/main/java/org/mifos/connector/channel/camel/routes/ChannelRouteBuilder.java

```diff
@@ -461,6 +433,44 @@ private void collectionRoutes() {
                 });
     }
 
+    private String getFinalAmsValue(JSONArray payer, Map<String, Object> extraVariables) {
+        String primaryIdentifierVal = "";
+        String secondaryIdentifierVal = "";
+        String primaryIdentifierName = "";
+        String secondaryIdentifierName = "";
+        String finalAmsVal = "value";
+        primaryIdentifierName = ((JSONObject) payer.get(0)).getString("key");
+        secondaryIdentifierName = ((JSONObject) payer.get(1)).getString("key");
+        primaryIdentifierVal = ((JSONObject) payer.get(0)).getString("value");
+        secondaryIdentifierVal = ((JSONObject) payer.get(1)).getString("value");
+        for (AMSProps.AMS amsIdentifier : amsUtils.postConstruct()) {
+            logger.info("KEY VALUE PAIR : " + amsIdentifier.getIdentifier() + " " + amsIdentifier.getValue());
```

Comment:
use {} to concatenate logger statements. and use debug-level logging for printing